README.md

```diff
@@ -224,7 +224,8 @@ Please see our [troubleshooting guide](https://github.com/sendgrid/sendgrid-pyth
 <a name="about"></a>
 # About
 
-sendgrid-python is guided and supported by the SendGrid [Developer Experience Team](mailto:dx@sendgrid.com).
+sendgrid-python is guided and supported by the SendGrid Developer Experience Team.
```

Comment:
Please add a space between these two lines. Thanks!

Reply:
Fixed spacing and alphabetical order